code/modules/events/ion_storm.dm

```diff
@@ -178,13 +179,29 @@ ABSTRACT_TYPE(/datum/ion_category)
 		if (!length(targets))
 			build_targets()
 		for (var/i in 1 to amount)
-			var/object = pick(targets)
+			var/atom/object = pick(targets)
+
+			//spatial interdictor: shield general hardware from ionic interference. law racks explicitly omitted due to sensitivity (and gameplay fun)
+			//consumes cell charge per hardware item protected, based on the category's interdict cost
+			var/interdicted = FALSE
+			for (var/obj/machinery/interdictor/IX in by_type[/obj/machinery/interdictor])
+				if (IN_RANGE(IX,object,IX.interdict_range) && IX.expend_interdict(interdict_cost))
+					interdicted = TRUE
+					SPAWN(rand(1,8))
+						playsound(object.loc, "sparks", 60, 1) //absorption noise, as a little bit of "force feedback"
+					//in case people say a thing was ion stormed but it actually was not because of interdiction
```

Comment:
I don't think this is really necessary. Targets are already picked randomly so this is just equivalent to the target not getting picked.
```suggestion
```

Reply:
Removed in [c923f6c](https://github.com/goonstation/goonstation/pull/11870/commits/c923f6ce9d6d7c4835c7dd74ce7dab91928a5230).